conformance/utils/http/http.go

```diff
@@ -83,9 +83,13 @@ type ExpectedRequest struct {
 
 // Response defines expected properties of a response from a backend.
 type Response struct {
-	StatusCode    int
-	Headers       map[string]string
-	AbsentHeaders []string
+	StatusCode       int
+	Headers          map[string]string
+	AbsentHeaders    []string
```

Comment:
@zhaohuabing you have to run `gofmt ./...` in order to fix the white spaces:
```suggestion
	StatusCode    int
	Headers       map[string]string
	AbsentHeaders []string
```
(https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/kubernetes-sigs_gateway-api/3739/pull-gateway-api-verify/1928279751638650880#1:build-log.txt%3A475)